the-exterminators/error-annihilator
# Sp 11 login authification context with db

## Linked Issues

## PR Description

## Changes


## How to test

1. Fetch Branch
2. Open, build and run project
3. Check if project builds and looks correct on first sight
4. Try to log in with the user 'admin' and the password 'password'
5. You should be able to login with the user (look at the logout button, it should say the username)
6. now log out and try to log in with the user 'burnsjana' and the password 'Louist91!' - you should be able to log in
7. The log out button and the my profile page should now have the username burnsjana

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
